src/ProjectNeon/Assets/Enemies/EncounterBuilder.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+public class EncounterBuilder : MonoBehaviour
+{
+    [SerializeField]
+    private List<Enemy> possible;
+
+    [SerializeField]
+    private int difficulty;
+
+    public List<Enemy> generate()
+    {
+        /**
+         * @todo #52:30min Implement encounter generation after the encounter algorithm is defined, and then
```

Comment:
Take at least a quick crack at making a basic algorithm here. Maybe just 5 minutes of work. 

If YOU were playing an RPG, what enemies would you expect to face?

Change this puzzle to Evolve Encounter Generation after playtesting. We want this Encounter Builder to work, even if it's really basic.